templates/miq-template.yaml

```diff
@@ -19,7 +19,8 @@ objects:
     name: "${NAME}-secrets"
   stringData:
     pg-password: "${DATABASE_PASSWORD}"
-    database_url: "postgresql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_SERVICE_NAME}/${DATABASE_NAME}?encoding=utf8&pool=5&wait_timeout=5"
+    database-url: "postgresql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_SERVICE_NAME}/${DATABASE_NAME}?encoding=utf8&pool=5&wait_timeout=5"
+    v2-key: "${V2_KEY}"
```

Comment:
@carbonin Minor, but I wonder if the key name and ENV var name should not have "v2" in them.  If we have to bump the key format, it will go to v3, and require all of this to change.  Not sure on a better name, but maybe `manageiq-encryption-key`?